File: frontend/src/types/owner.ts
export interface Owner {
  id: string;
  name: string;
  address: string;
  phone: string;
  birthday: string;
}

export interface OwnerCreate {
  name: string;
  address: string;
  phone: string;
  birthday: string;
}

export interface OwnerUpdate {
  name?: string;
  address?: string;
  phone?: string;
  birthday?: string;
}

File: backend/RealEstate.Domain/Entities/Owner.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace RealEstate.Domain.Entities
{
    public class Owner
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; } = string.Empty;

        [BsonElement("name")]
        public string Name { get; set; } = string.Empty;

        [BsonElement("address")]
        public string Address { get; set; } = string.Empty;

        [BsonElement("phone")]
        public string Phone { get; set; } = string.Empty;

        [BsonElement("birthday")]
        public DateTime Birthday { get; set; }
    }
}